### PR TITLE
tests: Fixed some unexpected errors

### PR DESCRIPTION
**Pull request by locke-lunarg**

### CODE REVIEW

File: tests/layer_validation_tests.cpp [hunk 1]
```diff
@@ -21547,9 +21581,8 @@ TEST_F(VkLayerTest, CreatePipelineTessErrors) {
 TEST_F(VkLayerTest, CreatePipelineAttribBindingConflict) {
     TEST_DESCRIPTION(
         "Test that an error is produced for a vertex attribute setup where multiple bindings provide the same location");
-    m_errorMonitor->SetDesiredFailureMsg(VK_DEBUG_REPORT_ERROR_BIT_EXT,
-                                         "Duplicate vertex input binding descriptions for binding 0");
-
+    // m_errorMonitor->SetDesiredFailureMsg(VK_DEBUG_REPORT_ERROR_BIT_EXT,
```
Comment (mark-lunarg) [inline tests/layer_validation_tests.cpp]: @locke-lunarg, are these commented-out lines replaced by the 00616 error below?  If so please remove these old lines.

File: tests/layer_validation_tests.cpp [hunk 2]
```diff
@@ -25415,6 +25450,7 @@ TEST_F(VkLayerTest, CopyImageMultiplaneAspectBits) {
     copy_region.srcOffset = {0, 0, 0};
     copy_region.dstOffset = {0, 0, 0};
 
+    m_errorMonitor->SetDesiredFailureMsg(VK_DEBUG_REPORT_ERROR_BIT_EXT, "VUID-vkCmdCopyImage-srcImage-00135");
```
Comment (mark-lunarg) [inline tests/layer_validation_tests.cpp]: Do we really want these new 'Desired' failure messages, or are they just popping up due to a bad test?  If we absolutely cannot prevent these messages when provoking the error we're looking for,  maybe we should use 'SetUnexpectedError'.

File: tests/layer_validation_tests.cpp (hunk 1 above)
Comment (locke-lunarg) [reply tests/layer_validation_tests.cpp]: Fixed it.

File: tests/layer_validation_tests.cpp (hunk 2 above)
Comment (locke-lunarg) [reply tests/layer_validation_tests.cpp]: Fixed it.

**Pull request merged**